lib/pymedphys/_experimental/chartchecks/weekly_check_helpers.py

```diff
@@ -28,62 +28,64 @@
 
 
 def show_incomplete_weekly_checks():
-    with connect.connect("PRDMOSAIQIWVV01.utmsa.local") as cursor:
-        incomplete = get_incomplete_qcls(cursor, "Physics Resident")
-        todays_date = date.today() + timedelta(days=3)
-        todays_date = todays_date.strftime("%b %d, %Y")
-        # todays_date = "Dec 4, 2020"
-        incomplete = incomplete[
-            (incomplete["task"] == "Weekly Chart Check")
-            & (incomplete["due"] == todays_date)
-        ]
-        incomplete = incomplete.drop(columns=["instructions", "task", "due", "comment"])
-        incomplete = incomplete.reset_index(drop=True)
+    cursor = _pp_mosaiq.connect("PRDMOSAIQIWVV01.utmsa.local")
```

Comment:
@rembishj @SimonBiggs are you intending to have a hardcoded server name here?

Reply:
It's certainly now how it needs to be in the long term. But we have a bit of a tiered quality bar within PyMedPhys. Given this is scoped under the 'experimental" package essentially "anything goes that doesn't break the build".

I am hoping to at some point go through a video call with @rembishj and show him the ropes on how to utilise site configuration for these sorts of things. But, for the time being he has other PhD demands that are taking precedence. For now I'm happy if he's getting something to work that works within his centre.

Also, it's not just @rembishj, I too have some experimental items where I have (for now) left the configuration hard coded:

https://github.com/pymedphys/pymedphys/blob/5c10f54d167f9e6416a60acbb300275d3deb60cd/lib/pymedphys/_experimental/streamlit/apps/dashboard.py#L30-L41

Reply:
Yep, what Simon said. I would like to make it more universal, but just trying to get it working first and then focus on that.